backend/blog/admin.py

```diff
@@ -1,3 +1,50 @@
 from django.contrib import admin
+from blog.models import Post, Tag, Profile
```

Comment:
Alternativ kannst du auch einfach `from .models import ...` benutzen, also den relativ import, weil du eh in demselben App bist, hier up to you.

Dabei was die Imports im Django generell angeht: es gibt eine bestimmte Reihenfolge, die mehr oder weniger so aussieht:

1. Imports aus den internen Packages / Modulen, die direkt vom Python kommen, z.B.:
   ```
   from collections import OrderedDict
   from typing import Optional
   ```
2. Imports aus den externen Packages:
   `from rest_framework import serializers`
3. Imports aus diversen Django Modulen:
   ```
   from django.contrib import admin
   from django.utils import timezone
   ```
4. Imports aus den anderen Apps im Rahmen deines Django-Projects:
   ```
   from some_app_name.models import SomeModel
   from another_app_name.services import SomeService
   ```
   etc.
5. Und am Ende kommen die Imports aus demselben App, wo du grade bist, also in dem Fall von dir:
   `from .models import Post, Profile, Tag` (generell in der alphabetischen Reihenfolge)

Nach jedem Import-Layer kommt eine leere Zeile. 

Dabei ganz Richtig ist, dass du hier aus den `blog.models` die Modellen einzeln importiert hast und nicht als `from blog.models import *` -> das ist tatsächlich best practices, sehr nice.